x-pack/legacy/plugins/maps/public/connected_components/map/mb/draw_control/draw_circle.js

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Please convert new module to TypeScript

Reply:
Can we avoid typescripting this file? There are no types for draw mode stuff and this is really just module for draw_control. Just imagine this file as an inline npm module without the annoyance of hosting in npm

Reply:
Thanks for the TS conversion. I merged these changes. lots of ts-ignores but good enough